metrics/web/test/common/presentation/metrics_theme/model/login_theme_data_test.dart

```diff
@@ -14,6 +14,7 @@ void main() {
         const themeData = LoginThemeData();
 
         expect(themeData.loginOptionButtonStyle, isNotNull);
+        expect(themeData.passwordVisibilityIconColor, isNotNull);
```

Comment:
Let's move this to the separate test and pass `null` in constructor as a value for this color.